frontend/app/component/form/candidates_votes_form/CandidatesVotesForm.tsx

```diff
@@ -125,6 +125,13 @@ export function CandidatesVotesForm({ group }: CandidatesVotesFormProps) {
   const hasValidationError = errors.length > 0;
   const hasValidationWarning = warnings.length > 0;
 
+  const defaultProps = {
+    errorsAndWarnings: isSaved ? errorsAndWarnings : undefined,
+    warningsAccepted: getIgnoreWarnings(),
+    inputProps: register(),
```

Comment:
was doubting if this would work with register as well but if all tests run :D

Reply:
Yeah I just tried it and it worked haha